src/lib/api-routes/outbound.js

```diff
@@ -85,6 +85,32 @@ router.post('/template/:traceID', [
   }
 })
 
+// TODO: Refactor the following. Decision to be taken whether to mix it up with actual template endpoint using query params for iternations
```

Comment:
Do we need a story for this?

Reply:
Removed that comment. I think the extra endpoint is better for this purpose.